app/templates/components/component-tree-item.hbs

```diff
@@ -21,14 +20,46 @@
   {{/if}}
 
   <code
-    class="component-tree-item__tag {{if @item.isComponent "component-tree-item__bracket"}} {{if (not @item.hasChildren) "component-tree-item__bracket--self-closing"}}"
+    class="component-tree-item__tag
+      {{if
+        (and (not @item.args.positional) @item.isComponent)
+        "component-tree-item__bracket"
+      }}
+      {{if
+        (and (not @item.args.positional) (not @item.hasChildren))
+        "component-tree-item__bracket--self-closing"
+      }}
+      {{if
+        (and @item.args.positional @item.isComponent)
+        "component-tree-item-classic__bracket"
+      }}"
   >
     {{#if @item.isComponent}}
-      {{classify @item.name}}
+      {{#if @item.args.positional}}
+        {{@item.name}}
+
+        {{#each @item.args.positional as |value|}}
+          {{component-argument value}}
+        {{/each}}
+
+        {{#each-in @item.args.named as |name value|}}
+          {{name}}={{component-argument value}}
+        {{/each-in}}
+      {{else}}
+        {{classify @item.name}}
+
+        {{#each-in @item.args.named as |name value|}}
+          {{#if (is-string value)}}
+            @{{name}}={{component-argument value}}
+          {{else}}
+            @{{name}}=\{{ {{~component-argument value~}}}}
+          {{/if}}
+        {{/each-in}}
+      {{/if}}
     {{else if @item.isOutlet}}
-      \{{outlet "{{@item.name}}"}}
+      {{outlet "{{@item.name}}"}}
```

Comment:
We should probably use your `component-argument` helper here (after you fixed to use `JSON.stringify`) since it has the same problem. Also the `\` is needed? Otherwise it will invoke `{{outlet}}` instead of rendering the string.